canonical-websites/www.canonical.com
# Add dockerfile

## Done

TO MERGE WHEN https://github.com/canonical-websites/www.canonical.com/pull/247 
* Add dockerfile to project 

## QA

1. Check out this feature branch
2. ` docker build --build-arg COMMIT_ID=test --tag prod-comms.docker-registry.canonical.com/www.canonical.com .`
3. `docker run prod-comms.docker-registry.canonical.com/www.canonical.com`
